### DISCUSSION TITLE
Problem with API

### DISCUSSION THREAD

**Discussion by alkura** [community/community#69309] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

I used JSON type for check data type from

**Comment by fubus05** [comment]
So use this link https://jsonplaceholder.typicode.com/todos

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
